src/Tizen.NUI/src/internal/Application.cs

```diff
@@ -1194,12 +1194,21 @@ public static Application New(int argc, string stylesheet, Application.WindowMod
 
         public static Application New(string[] args, string stylesheet, Application.WindowMode windowMode)
         {
-            int argc = args.Length;
-            string argvStr = string.Join(" ", args);
-
-            Application ret = new Application(NDalicPINVOKE.Application_New__MANUAL_4(argc, argvStr, stylesheet, (int)windowMode), true);
+            Application ret = null;
+            try
+            {
+                int argc = args.Length;
+                string argvStr = string.Join(" ", args);
 
-            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+                ret = new Application(NDalicPINVOKE.Application_New__MANUAL_4(argc, argvStr, stylesheet, (int)windowMode), true);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            }
+            catch (Exception exception)
+            {
+                Tizen.Log.Fatal("NUI", "message : " + exception.Message);
```

Comment:
how about to add some message more? 
ex: Tizen.Log.Fatal("NUI", "[Error] got exception during Application New(), this should not occur, msg : " + exception.Message);

Reply:
Fixed